src/SprykerEco/Yves/Computop/Mapper/Init/PostPlace/PayPalMapper.php

```diff
@@ -156,4 +171,60 @@ protected function getDataSubArray(ComputopPayPalPaymentTransfer $computopPayPal
 
         return $dataSubArray;
     }
+
+    /**
+     * @param \Generated\Shared\Transfer\ItemTransfer $item
+     *
+     * @return string
+     */
+    protected function getItemOrderDescription(ItemTransfer $item): string
+    {
+        return implode(',', [
+            $item->getName(),
+            $item->getUnitGrossPrice() - $item->getUnitTaxAmount(),
+            $item->getSku(),
+            $item->getQuantity(),
+            '',
+            $item->getUnitTaxAmount(),
+        ]);
+    }
+
+    /**
+     * Order item description for items should start from OrderDesc2 to OrderDesc99
+     *
+     * @param int $key
+     *
+     * @return string
+     */
+    protected function getOrderItemDescKey(int $key): string
+    {
+        return ComputopApiConfig::ORDER_DESC . ($key + 2);
```

Comment:
* Change to the `ComputopApiConfig::ORDER_DESC .$key++`.
* Create a separate variable outside the cycle to hold the index that starting from 2.